app/src/main/java/org/jellyfin/mobile/ApplicationModule.kt

```diff
@@ -15,4 +21,14 @@ val applicationModule: Module = module {
     single { OkHttpClient() }
     single(named(PLAYER_EVENT_CHANNEL)) { Channel<PlayerEvent>() }
     single(named(WEBAPP_FUNCTION_CHANNEL)) { Channel<String>() }
+    single {
+        Jellyfin {
```

Comment:
See https://github.com/jellyfin/jellyfin-apiclient-java/blob/master/android/src/main/kotlin/org/jellyfin/apiclient/JellyfinAndroid.kt

It will set the http client and server discovery